src/custom-element.js

```diff
@@ -30,7 +31,7 @@ import {ResourceState} from './service/resource';
 import {Services} from './services';
 import {Signals} from './utils/signals';
 import {blockedByConsentError, isBlockedByConsent, reportError} from './error';
-import {createLoaderElement} from '../src/loader';
+import {createLegacyLoaderElement} from '../src/loader-legacy';
```

Comment:
Should we keep these in the same file? It would avoid file renaming after launch which is annoying for code history.

Reply:
I personally prefer separate files to separate concerns, makes switching back and forth between two implementations easier. Git should preserve history for renames like this on its own, even without you explicitly `git mv`ing it.

Reply:
I tried to git blame a very old function recently and navigating file renames wasn't fun, at least in GitHub UI. Is there a more efficient technique that you know of?

Reply:
since there isn't much in `legacy-loader.js`, I will merge them and rename back to `loader.js`

Reply:
If I go to the blame for the loader-legacy file in this change ( https://github.com/ampproject/amphtml/blame/1e3c249a6e08097de908e4663080b6f7d324f2c3/src/loader-legacy.js), and click on "view blame prior to this change" on any line, it goes to the old file correctly.